frontend/benefit/applicant/browser-tests/roles/company.ts

```diff
@@ -0,0 +1,18 @@
+import { Role } from 'testcafe';
+import login from '../page-modal/login';
+import { getFrontendUrl } from '../utils/url.utils';
+
+const url = getFrontendUrl('/');
+
+const company = Role(url, async (t) => {
```

Comment:
Role seems to be a clean way to do the logging! I would suggest you to use also MOCK_FLAG (env variable) to completely mock the login from backend, it makes testing and developing easier. Not sure does your backend support it however...
Perhaps I could start using this roles as well.

Reply:
I added the MOCK flag and disabled the authentication, it worked well in the local env, but I am getting CORS errors I couldn't manage to work around.